### DISCUSSION TITLE
How to add a label to an issue in GitHub mobile?

### DISCUSSION THREAD

**Discussion by Ayfri** [community/community#22064] [New to GitHub]
<p>Hi, I’m using GitHub on my phone but I see no buttons or any options to add a label to an (existing) issue, how can I do it ? As I can’t go on any computer during 3 weeks it’s important for me.</p>

**Comment by ghost** [comment]
<p>Hey <a class="mention" href="/ayfri">@Ayfri</a>!</p>
<p>Thanks for posting your question to the Github Support Community.</p>
<p>To add a label to an issue using the GitHub app, simply:</p>
<ol>
<li>Go to the issue in-app,</li>
<li>Drag up the context menu from the bottom,</li>
<li>Tap <code>EDIT</code> next to the <code>Labels</code> header,</li>
<li>Change around the labels as you need,</li>
<li>Tap <code>SAVE</code>
</li>
</ol>
<p>If you run into any problems, just let me know what’s happened, what device and operating system you’re using, and the version of the GitHub app (Go to the <code>Home</code> page &gt; Tap your profile icon &gt; Tap the cog icon in the top-right &gt; Scroll down for the version number).</p>

**Reply by StephCurry07** [reply]
But there isn't an option to create a new label and add.
We can only assign the pre-existing labels.
Requesting GitHub community to provide such an option.

**Comment by Ayfri** [comment]
<p>Sorry but I don’t see any <code>EDIT</code> button for the labels.<br>
It’s not an issue from a repo that I own, I think that’s why it doesn’t work.!<br>
I’m using the GitHub app version 1.2.8 on Android 9<br>
<a href="https://cdn.discordapp.com/attachments/656977854986321941/737969225204760596/Screenshot_20200729-114556.png" rel="nofollow noopener">Image</a></p>

**Comment by Ayfri** [comment, marked as answer]
<p>Oh I’m stupid, we can’t add labels on an issue on a repo that we don’t own :man_facepalming:</p>
